core/string_api.php

```diff
@@ -666,7 +667,15 @@ function string_get_bug_view_link( $p_bug_id, $p_detail_info = true, $p_fqdn = f
 				$t_link .= ' class="resolved"';
 			}
 		}
-		$t_link .= '>' . bug_format_id( $p_bug_id ) . '</a>';
+		$t_link .= '>' . bug_format_id( $p_bug_id );
+		if ( $p_long_link ) {
+			$t_short_summary = $t_summary;
```

Comment:
This line should be removed as the value of $t_short_summary is not used before the next assignment of $t_short_summary.